### PR TITLE
Initial GitHub markdown update

### PR DESCRIPTION
**Pull request by aFinchy**
Added markdown Document. 
Added links to this document in the main readme on the repository. 

This document is created to assist anyone who may be looking to update existing files on GitHub or create new one. It will help SIG chairs keep their repositories up-to-date and use the proper formatting along with show the types available. 

Signed-off-by: Amy Finch <amy@finchstudio.ca>

### CODE REVIEW

File: markdown/github-markdown/README.md [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+  <dd>This is another definition of the second term.</dd>
+</dl>
+
+```html
+<p>Markdown in HTML does *not* work **well**. Use <i>HTML</i> <b>tags</b> instead.</p>
+```
+
+<p>Markdown in HTML does *not* work **well**. Use <i>HTML</i> <b>tags</b> instead.</p>
+
+## Colapsable section
```
Comment (chanmosq) [inline markdown/github-markdown/README.md:363]:
```suggestion
## Collapsible section
```

File: markdown/github-markdown/README.md [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+
+## Link to the top of the page or a specific section on said page
+
+[Go To TOP](#Table_of_contents)
+<a name="Table_of_contents"></a>
+   
+```markdown
+[Go To TOP](#Table_of_contents)
+<a name="Table_of_contents"></a>
+```
```
Comment (chanmosq) [inline markdown/github-markdown/README.md:391]: The link doesn't work as is, it must be changed to dashes. I also suggest lower-casing to show that it's not case-sensitive
````suggestion
[Go To TOP](#table-of-contents)
<a name="table-of-contents"></a>
   
```markdown
[Go To TOP](#table-of-contents)
<a name="table-of-contents"></a>
``` 
````

File: markdown/github-markdown/README.md [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+<a name="Table_of_contents"></a>
+   
+```markdown
+[Go To TOP](#Table_of_contents)
+<a name="Table_of_contents"></a>
+``` 
+
+```markdown
+[a Name](#Section_ID)
+<a name="Section_ID></a>
```
Comment (chanmosq) [inline markdown/github-markdown/README.md:395]:
```suggestion
[a Name](#Section-ID)
<a name="Section-ID></a>
```

**Review by chanmosq — APPROVED** [review]
Made a few corrections, otherwise looks good! Thanks a lot for adding this in.

**Comment by aFinchy** [comment]
Thanks @chanmosq, applied all the changes.

**Pull request merged**